README.adoc

```diff
@@ -3,34 +3,10 @@
 
 This is the official Gradle plugin to compile Frege projects (https://github.com/Frege/frege).  See the example project, https://github.com/mperry/frege-gradle-example, for how to use this plugin.
```

Comment:
This example project should be in CI and should probably be either an official Frege project or part of this repo.